### PR TITLE
Document the behavior when creating a new env using an existing env name

### PR DESCRIPTION
**Pull request by lee3star**
Describe an un-common behavior/feature when creating an environment when an existing environment name is used. In this case, instead of returning 400 or 409 as normal RESTful service does, the call/operation is successful with a new environment created. The new environment has a unique ID, but the name is not the name you entered, but a "-<n>" appended to it, the <n> is the incremental number. This is working as designed as per the conversation here: https://mulesoft.slack.com/archives/G7JFWLX54/p1598918390006400

### CODE REVIEW

File: modules/ROOT/pages/environments.adoc [hunk 1]
```diff
@@ -38,7 +38,8 @@ To create a new environment or manage an existing environment:
 image::environments-85dc3.png[]
 
 [NOTE]
-Depending on your CloudHub subscription, you may have restrictions on the number and type of environments you can create.
+* Depending on your CloudHub subscription, you may have restrictions on the number and type of environments you can create.
+* If an existing name is used to create a new environment, the newly created evironment name will have a "-<n>" appended to it, where the <n> is an incremental number
```
Comment (sarathecoubian) [inline modules/ROOT/pages/environments.adoc]:
```suggestion
* If you create a new environment using the name of an existing environment, the newly created environment name has `-<n>` appended to it, where `<n>` is an incremental number.
```

Comment (sarathecoubian) [reply modules/ROOT/pages/environments.adoc]: @lee3star Please implement this change, and I can approve and merge when ready.

**Review by sarathecoubian — DISMISSED** [review]

File: modules/ROOT/pages/environments.adoc [hunk 2]
```diff
@@ -38,7 +38,8 @@ To create a new environment or manage an existing environment:
 image::environments-85dc3.png[]
 
 [NOTE]
-Depending on your CloudHub subscription, you may have restrictions on the number and type of environments you can create.
+* Depending on your CloudHub subscription, you may have restrictions on the number and type of environments you can create.
```
Comment (sarathecoubian) [inline modules/ROOT/pages/environments.adoc]:
```suggestion
* Depending on your CloudHub subscription, you might have restrictions on the number and type of environments you can create.
```

File: modules/ROOT/pages/environments.adoc [hunk 3]
```diff
@@ -38,7 +38,8 @@ To create a new environment or manage an existing environment:
 image::environments-85dc3.png[]
 
 [NOTE]
-Depending on your CloudHub subscription, you may have restrictions on the number and type of environments you can create.
+* Depending on your CloudHub subscription, you might have restrictions on the number and type of environments you can create.
+* If you create a new environment using the name of an existing environment, the newly created environment name has `-<n>` appended to it, where `<n>` is an incremental number.
```
Comment (dukesphere) [inline modules/ROOT/pages/environments.adoc]:
```suggestion
* If you create a new environment using the name of an existing environment, `-<n>` is appended to the name of the new environment. `<n>` is an incremental number.
```

**Review by dukesphere — DISMISSED** [review]
suggested change

**Review by dukesphere — APPROVED** [review]

**Pull request merged**